bcda-site-static/user_guide.md

```diff
@@ -624,3 +624,164 @@ The response will be the requested data as FHIR Patient resources in NDJSON form
+
+  ###### Headers
+	* `Content-Location: https://sandbox.bcda.cms.gov/api/v1/jobs/44`
+
+  #### 3. Check the status of the export job
+
+  ##### Request
+	`GET https://sandbox.bcda.cms.gov/api/v1/jobs/44`
+
+  Using the `Content-Location` header value from the Coverage data export response, you can check the status of the export job. The status will change from `202 Accepted` to `200 OK` when the export job is complete and the data is ready to be downloaded.
+
+  ###### Headers
+	* `Authorization: Bearer {token}`
+
+  ###### cURL Command
+	```sh
+	curl -v https://sandbox.bcda.cms.gov/api/v1/jobs/44 \
+	-H 'Authorization: Bearer {token}'
+	```
+
+  ##### Responses
+	* `202 Accepted` indicates that the job is processing. Headers will include `X-Progress: In Progress`
+	* `200 OK` indicates that the job is complete. Below is an example of the format of the response body.
+  ```json
+  {
+	  "transactionTime": "2018-10-19T14:47:33.975024Z",
+	  "request": "https://sandbox.bcda.cms.gov/api/v1/Coverage/$export",
+	  "requiresAccessToken": true,
+	  "output": [
+	    {
+	      "type": "Coverage",
+	      "url": "https://sandbox.bcda.cms.gov/data/44/DBBD1CE1-AE24-435C-807D-ED45953077D3.ndjson"
+	    }
+	  ],
+	  "error": [
+	    {
+	      "type": "OperationOutcome",
+	      "url": "https://sandbox.bcda.cms.gov/data/44/DBBD1CE1-AE24-435C-807D-ED45953077D3-error.ndjson"
+	    }
+	  ]
+  }
+  ```
+  Coverage demographic data can be found at the URLs within the `output` field. The number 44 in the data file URLs is the same job ID from the `Content-Location` header URL in previous step. If some of the data cannot be exported due to errors, details of the errors can be found at the URLs in the `error` field. The errors are provided in an NDJSON file as FHIR [OperationOutcome](https://www.hl7.org/fhir/operationoutcome.html) resources.
+
+  #### 4. Retrieve the NDJSON output file(s)
+	To obtain the exported coverage data, a GET request is made to the output URLs in the job status response when the job reaches the Completed state. The data will be presented as an NDJSON file of [Coverage](https://www.hl7.org/fhir/coverage.html) resources.
+
+  ##### Request
+	`GET https://sandbox.bcda.cms.gov/data/44/DBBD1CE1-AE24-435C-807D-ED45953077D3.ndjson`
+
+  ###### Headers
+	* `Authorization: Bearer {token}`
+
+  ###### cURL command
+	```sh
+	curl https://sandbox.bcda.cms.gov/data/44/DBBD1CE1-AE24-435C-807D-ED45953077D3.ndjson \
+	-H 'Authorization: Bearer {token}'
+	```
+
+  ##### Response
+	The response will be the requested data as FHIR Coverage resources in NDJSON format. An example of one such resource is shown below.
+
+  ```json
+  {
+  "fullUrl": "https:///v1/fhir/Coverage/part-a-19990000002901",
```

Comment:
Is there supposed to be a domain in this URL?

Reply:
I have no idea

Reply:
Looks like it is the same for Patient and EOB on sandbox.bcda.cms.gov:
```
"fullUrl": "https:///v1/fhir/Patient/19990000002901",
```